pyturbo/systems/nozzle/nozzle_aero.py

```diff
@@ -87,17 +98,45 @@ def compute(self):
         self.fl_out.Pt = self.fl_in.Pt
         self.fl_out.Tt = self.fl_in.Tt
 
+        self.Ps_crit = ((2 / (self.gamma + 1)) ** (self.gamma / (self.gamma - 1))) * self.fl_out.Pt
+        self.Ps1 = self.fl_in.Pt - 0.5 * ((self.fl_in.W**2) / (self.rho_1 * (self.area_in**2)))
+        self.Ps2 = max(self.Ps_crit, self.pamb)
```

Comment:
see gas thermo librairy each time it is possible

Reply:
Code has been modified to use thermo library functions are used whe possible